src/common/level.ts

```diff
@@ -0,0 +1,21 @@
+export enum Level {
+  LOW = 'LOW',
+  MEDIUM = 'MEDIUM',
+  HIGH = 'HIGH',
+  UNKNOWN = 'UNKNOWN',
+}
```

Comment:
I'd still be a fan of removing the string values and just make this:
```
export enum Level { LOW, MEDIUM, HIGH, UNKNOWN }
```

You resolved my comment without replying so not sure if you left this as-is intentionally (which is fine, if so) or by accident.

Reply:
yeah i thought it was gonna take work to fix but i changed it to .name from the enum and that should fix it easily in the place it was using the values